src/database/ProductTransactions/index.ts

```diff
@@ -0,0 +1,98 @@
+import dbConnection, { tableNames, execute } from '../connection';
+
+const R = require('ramda');
+
+const productTransactionsTable = () => dbConnection()(tableNames.PRODUCT_TRANSACTIONS);
+const productTransactionsAttributesTable = () => dbConnection()(tableNames.PRODUCT_TRANSACTION_ATTRIBUTES);
+
+/**
+ * Represents a product transaction, except for it's typeid
+ */
+interface ProductTransactionDb {
+  producttransactionuuid: string;
+  productunits: string;
+  datetime: string;
+  topersonuuid: string;
+  frompersonuuid: string;
+  amountofproduct: number;
+  costperunit: number;
+  currency: string;
+  lastmodified: string;
+}
+
+interface ProductTransactionAttribute {
+  producttransactionuuid: string;
+  attrname: string;
+  attrvalue: string;
+}
+
+interface ProductTransaction {
+  uuid: string;
```

Comment:
i just realized we aren't specifying what type the product is. should we add that?